substrate-node/pallets/pallet-tft-price/src/lib.rs

```diff
@@ -366,8 +380,9 @@ impl<T: Config> Pallet<T> {
     }
 
     fn calc_avg() -> u32 {
-        let queue = Self::queue_transient();
-        let items = queue.get_all_values();
+        let items: Vec<u32> = TftPriceHistory::<T>::iter_values()
```

Comment:
I am looking for better solution but for now should be ok to avoid error messages (see #423)